### PR TITLE
Nitro Connect / GUI tweaks

### PR DESCRIPTION
**Pull request by geoknee**
* little bit of colour
* remove dud button
* make inbound/outbound/mediated channel type display differently 
Screenshot 2023-09-13 at 11 27 43

### CODE REVIEW

File: packages/nitro-gui/src/components/PaymentChannelContainer.tsx [hunk 1]
```diff
@@ -65,6 +79,18 @@ const PaymentChannelContainer: FC<Props> = ({
     }
   }, [nitroClient, ledgerChannel]);
 
+  const payer = focusedPaymentChannel.Balance.Payer;
+  const payee = focusedPaymentChannel.Balance.Payee;
+
+  const inferType = () => {
```
Comment (NiloCK) [inline packages/nitro-gui/src/components/PaymentChannelContainer.tsx:85]: Does this expect to be reused / exported? As it stands, could just be in-line code.

File: packages/nitro-gui/src/styles/colors.module.css [hunk 2]
```diff
@@ -8,4 +8,5 @@
   cGreen: #4caf50; /* mui green-500 */
   cAmber: #ffa000; /* mui amber-700 */
   cRed: #b71c1c; /* mui red-900 */
+  cBlue: aliceblue;
```
Comment (NiloCK) [inline packages/nitro-gui/src/styles/colors.module.css:11]: A natural choice.

**Review by NiloCK — APPROVED** [review]
This LGTM code-wise but there is no storybook preview.

Does https://nitro-storybook.netlify.app/ update with changes to `main`?

**Comment by geoknee** [comment]
> This LGTM code-wise but there is no storybook preview.

Yes that is unfortunate. Between the netlify dashboard and our `netlify.toml` config things aren't setup right now to generate one. I did manage to trigger one manually using a build hook:

https://gui-tweaks--nitro-storybook.netlify.app/?path=/story/paymentchanneldetails--payment-channel-details-component&args=t:0


It is somewhat awkward that we have several sites building against the same `netlify.toml` file -- they are all evaluating the same condition to see whether the build should be skipped or not. 
> 
> Does https://nitro-storybook.netlify.app/ update with changes to `main`?

Yes I believe it does.

**Pull request merged**